aim/ext/transport/config.py

```diff
@@ -11,3 +11,4 @@
 # GRPC OPTIONS
 AIM_RT_MAX_MESSAGE_SIZE = '__AIM_RT_MAX_MESSAGE_SIZE__'
 AIM_RT_DEFAULT_MAX_MESSAGE_SIZE = 16 * 1024 * 1024  # 16MB
+AIM_RT_BEARER_TOKEN = None
```

Comment:
I guess this should look like something like this: 
```
AIM_RT_BEARER_TOKEN = '__AIM_RT_BEARER_TOKEN__'
```

Reply:
Thank you for this feedback! I made the change you suggested.